ts/src/binance.ts

```diff
@@ -5866,6 +5858,7 @@ export default class binance extends Exchange {
         }
         let postOnly = undefined;
         if (!isPortfolioMargin) {
+            request['newOrderRespType'] = this.safeString (this.options['newOrderRespType'], type, 'RESULT'); // 'ACK' for order id, 'RESULT' for full order or 'FULL' for order with fills
```

Comment:
@Dan-krm portfolioMargin also supports this flag, so we can keep it the same for all of them 
![image](https://github.com/ccxt/ccxt/assets/43336371/8ae86af2-a902-4125-b6b8-66eb1f8fbdff)

Reply:
although full is not supported there